src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,39 @@
+package controller;
+
+import controller.dto.LottosDto;
+import controller.dto.StatisticDto;
+import domain.*;
+import service.LottoService;
+
+import java.util.List;
+
+public class LottoController {
+
+    private final LottoService lottoService;
+
+    public LottoController(LottoService lottoService) {
+        this.lottoService = lottoService;
+    }
+
+    public LottosDto purchase(int inputMoney, int manualLottoCount, List<String[]> manualLottoNumbers) {
+        Money money = lottoService.createMoney(inputMoney);
+        int autoLottoCount = lottoService.getAutoLottoCount(money, manualLottoCount);
+        Lottos lottos = lottoService.generateLottos(manualLottoNumbers, autoLottoCount);
+
+        return LottosDto.from(lottos, lottos.size());
+    }
+
+    public StatisticDto winningResult(String[] inputWinningNumber,
+                                      int inputBonusBall,
+                                      List<String[]> manualLottoNumbers,
```

Comment:
당첨 결과를 확인하는 모든 로또를 입력으로 받아야 하는 거 아닌가요!?
```suggestion
                                      List<String[]> lottos,
```

Reply:
아예 로또들 Dto를 받도록 수정했습니다! List<String[]>으로 하려면 만들어진 Lottos를 다시 돌면서 List<String[]> 에 맞는 형으로 바꿔야 할것 같아서 Dto로 넘겼습니다.

그런데 이렇게하니 `LottosDto`가 많이 무거워진 것 같아서 어떻게 해야 할지 고민입니다..